string objectName
string agentName
string locationName
string goal
---
bool answer